includes/discovery/processors/cambium.inc.php

```diff
@@ -21,7 +21,7 @@
             '0',
             'cambium-cpu',
             $descr,
-            '100',
+            '10',
```

Comment:
I do sometimes worry about changes like this as it could have been ok for earlier firmware for the device and now Cambium have changed it so this would break things.

@pheinrichs are you able to comment on this to check as you added the original support?

@neg2led if this is approved by pheinrichs then I think you can actually now get rid of the polling file as it shouldn't be needed.